osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -115,21 +119,25 @@ def extract_excerpts(execution_configuration, extraction_order, bbox_args, conve
                                 email=False
                             )
                         else:
+                            message = _('The extraction of "%(file)s" of extraction order "%(order_id)s" failed.') % {
+                                'file': result_file_name,
+                                'order_id': extraction_order.id
+                            }
```

Comment:
yes: we generally went over to using `.format`, and older code is still using `%`. Even though I move the code only around, I think  it is a good idea to use `.format` more consistently, thanks for the input.